src/revaultd/model.rs

```diff
@@ -100,13 +103,55 @@ impl std::fmt::Display for VaultStatus {
             Self::Canceled => write!(f, "Canceled"),
             Self::EmergencyVaulting => write!(f, "Emergency vaulting"),
             Self::EmergencyVaulted => write!(f, "Emergency vaulted"),
-            Self::Spendable => write!(f, "Spendable"),
+            Self::UnvaultEmergencyVaulting => write!(f, "Unvault Emergency vaulting"),
+            Self::UnvaultEmergencyVaulted => write!(f, "Unvault Emergency vaulted"),
             Self::Spending => write!(f, "Spending"),
             Self::Spent => write!(f, "Spent"),
         }
     }
 }
 
+impl VaultStatus {
+    pub const CURRENT: [VaultStatus; 11] = [
+        Self::Funded,
+        Self::Securing,
+        Self::Secured,
+        Self::Activating,
+        Self::Active,
+        Self::Unvaulting,
+        Self::Unvaulted,
+        Self::Canceling,
+        Self::EmergencyVaulting,
+        Self::UnvaultEmergencyVaulting,
+        Self::Spending,
+    ];
```

Comment:
What is the definition of a "Current" vault? One that is not yet entirely Moved? It's weird to me that Unvaulted vaults are both "Current" and "Moving".

If we want mutually exclusive sets maybe "Secured only" and "Delegated only"  as you previously had made sense.
If we want non-exclusive ones maybe just rename to "All" ? And add the Canceled and Spent one

Aaaarg it's hard.